dockerfile/correctness/multiple-entrypoint-instructions.yaml

```diff
@@ -6,15 +6,10 @@ rules:
       - pattern: |
           ENTRYPOINT ...
           ...
-          $ENTRYPOINT ...
-      - metavariable-pattern:
-          metavariable: $ENTRYPOINT
-          patterns:
-            - pattern: |
-                ENTRYPOINT ...
-      - focus-metavariable: $ENTRYPOINT
+          ENTRYPOINT ...
     message: >-
-      Multiple ENTRYPOINT instructions were found. Only the last one will take effect.
+      Multiple ENTRYPOINT instructions were found. Only the last one
+      will take effect.
```

Comment:
I don't know why the previous version was working because `$ENTRYPOINT ...` would parse as a shell command, not a dockerfile instruction. The new version is parsed and works as expected but the result is the whole region starting and ending with `ENTRYPOINT` and no longer just one `ENTRYPOINT` line.

Reply:
I had to look at the grammar definition to figure out what's going on :-|
What's expected to work is either a [metavariable representing a whole dockerfile instruction or an ellipsis representing a sequence of instructions](https://github.com/returntocorp/ocaml-tree-sitter-semgrep/blob/4374124a9fce73c9c6a8fc23d52edde7c985f259/lang/semgrep-grammars/src/semgrep-dockerfile/grammar.js#L17-L18).

The line `$ENTRYPOINT ...` confused me. It should probably be rejected the parser. The non-misleading way to write it is:
```
$ENTRYPOINT
...
```
i.e. `$ENTRYPOINT` captures the whole instruction `ENTRYPOINT /bin/true`, not just `ENTRYPOINT` like I thought. The ellipsis `...` is superfluous.

I pushed a revised version that restores the original behavior and clarifies what's going on.

Reply:
I pushed another change to make the solution closer to the original one (using a semgrep pattern instead of a regexp).